egs/TAC/run.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+set -o pipefail
+
+# Main storage directory where dataset will be stored
+storage_dir=$(readlink -m ./datasets)
+
+librispeech_dir=$storage_dir/LibriSpeech
+noise_dir=$storage_dir/Nonspeech
+CONF_FILE=./local/conf.yml
```

Comment:
This is unused. 
I understand that we would want to limit the confusion between conf args and bash args, but here, there is no way to have several config files.

Let's add an argument for that in the train file.

Reply:
added